web/blog/tenzir-node-v4.21/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Tenzir Node v4.21
+authors: [raxyte, IyeOnline]
+date: 2024-10-04
+tags: [release, node]
+comments: true
+---
+
+Parsing is now easier, faster, and better than before with [Tenzir Node
+v4.21][github-release]. Also: introducing an all-new integration with Azure Blob
+Storage.
+
+![Tenzir Node v4.21](tenzir-node-v4.21.excalidraw.svg)
+
+[github-release]: https://github.com/tenzir/tenzir/releases/tag/v4.21.0
+
+<!-- truncate -->
+
+## Reading in Fortinet and Sophos logs
+
+Tenzir now supports reading Fortinet and Sophos logs. This feature is powered
+by an update to the KV-Parser, which has been upgraded to allow quoted values.
+For example, here's how you can use the new parser to take apart a Fortinet
+FortiGate log file:
+
+```text {0} title="Parse a Fortinet log file"
+from "fortinet.log" read kv
+| parse rawdata kv "\|" "="
+```
```

Comment:
A before and after example is necessary to make the headline credible. One log sample suffices.